### ISSUE TITLE
Error in  elastic-beanstalk/deep-security/99deepsecurity-amzn1-x86_64.config.ebextension

### ISSUE THREAD

**Issue by fourninecloudsolutions** [deep-security/elastic-beanstalk#4]
Small syntax error in elastic-beanstalk/deep-security/99deepsecurity-amzn1-x86_64.config.ebextension file 
we need to remove  **`**  near check-certitcates (  /tmp/agent.rpm --quiet --no-check-certificate`')

**Comment by criscunhasantos** [comment]
Exact. At the end of line 3 there's a backtick that shouldn't be there.

**Comment by Makohan** [comment]
For activate on windows, file should be added `waitAfterCompletion` at last line?
